templates/registry/view_person.html

```diff
@@ -116,13 +116,18 @@ <h4>{{ person_details.first_name }} {{ person_details.surname }}
                                             </tr>
                                             <tr>
                                                 <td class="field">Date Of Birth : </td>
+                                                {% if person_details.age > '116 years'%}

```

Comment:
I dont know if this will work as expected. You are using a comparative operator on a string.

Reply:
Noted, please  Merge with master because there are other changes I made. I will have to check on it the above issue.